enterprise_access/apps/subsidy_access_policy/constants.py

```diff
@@ -79,3 +79,31 @@ class TransactionStateChoices:
     PENDING = 'pending'
     COMMITTED = 'committed'
     FAILED = 'failed'
+
+
+class MissingSubsidyAccessReasonUserMessages:
+    """
+    User-friendly display messages explaining why the learner does not have subsidized access.
+    """
+    ORGANIZATION_NO_FUNDS = "You can't enroll right now because your organization doesn't have enough funds."
+    ORGANIZATION_NO_FUNDS_NO_ADMIN_CONTACT = \
```

Comment:
[inform] This constant is currently unused and I'm intentionally deferring on its use for now. The future intent here is that if the enterprise customer does not have any enterprise administrators, the `ORGANIZATION_NO_FUNDS_NO_ADMIN_CONTACT` message should be returned; otherwise, the `ORGANIZATION_NO_FUNDS` message should be returned.

We would need to get the admin contacts for the enterprise from the LMS, i.e. with `get_enterprise_customer_data` in the `LmsApiClient` in order to support this.

Deferred for a future PR/ticket.